__test__/index.spec.mjs

```diff
@@ -0,0 +1,28 @@
+import test from 'ava'
```

Comment:
I don't really want to pull in third-party test frameworks. I never do in my projects, only the built-in assert module.

The old test.js is all that's needed, IMO. We don't care what Google's HTTP server returns, just that the TLS handshake completes successfully.